lib/panel/panel-manager.js

```diff
@@ -194,13 +180,15 @@ class PanelManager {
       return
     }
 
-    this.goPlusStatusBar = new GoPlusStatusBar({
+    const goPlusStatusBar = new GoPlusStatusBar({
```

Comment:
this code path is only called from main.js but the function there is not called. Should I remove it or actually use it, what do you think?

Reply:
Afaik this was only used to display some kind of busy indicator. It is not updated anywhere. I will remove it :)

Reply:
I think this was used to display the signature of the function under the cursor in the status bar.

This was powered by gocode, and I'm not sure if mdempsky/gocode even supports it anymore.  Plus, now that we have hover tooltips the future is less useful.   I'm all for removing it.